src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -0,0 +1,42 @@
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Admin page</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    
+    <a href="/conversations">Conversations</a>
+    <a href="/login">Login</a>
+    <a href="/about.jsp">About</a>
+  </nav>
+
+  <div id="container">
+    <h1>Admin Page</h1>
+
+    <% if(request.getAttribute("error") != null){ %>
+        <h2 style="color:red"><%= request.getAttribute("error") %></h2>
+    <% } %>
```

Comment:
What does this if statement do?

Reply:
I grabbed some code from register.jsp file, I'm pretty sure I removed it in my 2nd commit! It prints a header2 in red color with the specific error that happened! Try and register a new user that already exists and you'll see what happens :)

Reply:
Okay cool, I just tested it and I see now. Well everything else here makes perfect sense to me. I can also confirm that it was removed in that 2nd commit , no longer see it in the files changed section.